RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -116,3 +126,13 @@ def _extend_pixelLess(x):
     x.setsToMerge[0].tLists += [6]
 (trackingPhase2PU140 & vectorHits).toModify(earlyGeneralTracks, _extend_pixelLess)
 
+from Configuration.ProcessModifiers.trackingLST_cff import trackingLST
+(trackingPhase2PU140 & trackingLST).toModify(earlyGeneralTracks,
+                         TrackProducers = ['highPtTripletStepLSTpTracks', 'highPtTripletStepLSTT5Tracks'],
+                         hasSelector = [1,0],
+                         indivShareFrac = [0.1,0.1],
```

Comment:
`indivShareFrac` is a parameter that controls the level of duplicate cleaning within the same `Track` collection: the lower it is, the stronger the duplicate cleaning. From previous studies, we noticed that, even though LST has a low fake rate, it could profit from stronger duplicate cleaning, hence the lower values for LST tracks.